templates/deployment.yaml

```diff
@@ -541,9 +545,31 @@ spec:
           mountPath: /opt/bird/
         - name: bird-socket
           mountPath: /var/run/bird/
+      {{- if .Values.bird.birdConfigWatcher.enabled }}
+      - name: bird-config-watcher
+        image: {{ .Values.bird.birdConfigWatcher.image.repository }}:{{ .Values.bird.birdConfigWatcher.image.tag }}
+        imagePullPolicy: {{ .Values.bird.birdConfigWatcher.image.pullPolicy | quote }}
+        resources:
+{{ toYaml .Values.resources.birdConfigWatcher | indent 10 }}
+        securityContext:
+          capabilities:
+            add:
+            - NET_ADMIN
+            - SYS_ADMIN
+            - SETPCAP
+            - NET_RAW
+        ports:
+        - name: p8000
+          containerPort: 8000
+          protocol: TCP
+        volumeMounts:
+        - name: bird-config
+          mountPath: /opt/bird/
+        - name: bird-socket
+          mountPath: /var/run/bird/
       {{- end }}
 
-      {{- if and .Values.bird.enabled .Values.bird.birdExporter.enabled }}
+      {{- if .Values.bird.birdExporter.enabled }}
```

Comment:
Why would I want the bird-exporter if bird is not enbled?

Reply:
have a look at line 
https://github.com/jukrut/cgw/blob/feature/bird-reload/templates/deployment.yaml#L530
it will probably answer your questions.